src/main/resources/poc_pipeline.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
```

Comment:
is the poc intended to be in this PR? if so then it should be moved to the test resources and it might be nice to refer to it in the README.